blaze-core/src/main/scala/org/http4s/blazecore/Http1Stage.scala

```diff
@@ -26,6 +26,8 @@ trait Http1Stage[F[_]] { self: TailStage[ByteBuffer] =>
 
   protected implicit def F: Effect[F]
 
+  protected val chunkBufferMaxSize: Int
```

Comment:
Eager vals in a base class terrify me, but I think this one is safe because it's not referenced until it starts serving requests.  I don't think there's any harm in making this a def, though.